editor/filesystem_dock.h

```diff
@@ -101,7 +101,7 @@ class FileSystemDock : public VBoxContainer {
 
 private:
 	enum FileMenu {
-		FILE_OPEN,
+		FILE_OPEN_, // Conflict with WinAPI.
```

Comment:
Might be worth renaming this and the rest of the enum to something more verbose, like `FILE_MENU_OPEN`.

Reply:
> Or maybe it would work with enum class?

I think `FILE_OPEN` from windows has precedence no matter what you do to `FileMenu::FILE_OPEN`, since it's a preprocessor macro.
That's why you avoid making macros as a library, windows 😅

Reply:
> Might be worth renaming this and the rest of the enum to something more verbose, like `FILE_MENU_OPEN`.

I did this in #105924, it turned out to be quite a rabbit hole as I tried to put some more clarity in the names and ordering of the enum values too.